task_13/test/test.js

```diff
@@ -0,0 +1,45 @@
+const assert = require('chai').assert
+const {chef, Calculator} = require('../app')
+
+describe('Chef test', ()=>{
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/jkalandarov/epam_stage_2/pullRequest?prid=8887248)
```suggestion
describe('Chef test', () =>{
```